lessons/linear-regression/langs/en-us/pages/lr-plot-explore.adoc

```diff
@@ -0,0 +1,47 @@
+= lr-plot Explore
+
+**Part 1: You should already have plotted `lr-plot(animals-table, "name", "age", "weeks")` in the @starter-file{animals}... **
+
+@n What is the predictor function? @math{y =} @fitb{5em}{} @math{x +} @fitb{5em}{}
+
+@n What is the slope? @fitb{}{}
+
+@n What is the y-intercept? @fitb{}{}
+
+@n How long would our line of best fit predict it would take for a 5 year-old animal to be adopted? @fitb{}{@ifsoln{about 6 weeks}}
+
+@n What if they were a newborn, or just 0 years old? @fitb{}{@ifsoln{about 2 weeks}}
```

Comment:
If we're going to ask this question, we should also ask why this is answer is not to be trusted. 

The lesson text points out that we didn't have any animals with this age, and the implication there is that the y-intercept is not useful. We should remind them of that here.